app/lib/package/api_export/exported_api.dart

```diff
@@ -26,7 +26,7 @@ final _log = Logger('api_export:exported_bucket');
 /// This ensures that we don't delete files we've just created.
 /// It's entirely possible that one process is writing files, while another
 /// process is running garbage collection.
-const _minGarbageAge = Duration(days: 1);
+const _minGarbageAge = Duration(hours: 3);
```

Comment:
This shouldn't be high, if something is published, it'll be at-least 3 hours before we can delete it.

I think that's perhaps somewhat reasonable. But we don't want this to be too high.

However, 3 hour should be more than enough for any eventual consistency issues between datastore and GCS to have been resolved.